tests/manual_check/premodit/compare_with_direct3D.py

```diff
@@ -0,0 +1,60 @@
+"""This code compores premodit with direct computation in terms LSD of the line strength, ignoring the broadening parameters.
+
+"""
+import numpy as np
+import jax.numpy as jnp
+
+def compare_with_direct3D(mdb,nus,Ttest=1000.0,interval_contrast=0.1,Ttyp=2000.0):
+    """ compare the premodit LSD with the direct computation of LSD 3D version
+    
+    """
+    from exojax.spec.lsd import npgetix
+    from exojax.spec.hitran import SijT
+    from exojax.spec.premodit import make_elower_grid, unbiased_lsd
+
+    broadpar=np.array([mdb._n_Texp,mdb._alpha_ref]).T
+    uidx_broadpar, uniq_broadpar=uniqidx_2D(broadpar)
```

Comment:
uniqidx_2D is not defined (exojax.utils.indexing.uniqidx maybe?). 
Some other functions below such as npadd1D or make_lbd3D_uniqidx are also missing.
(Do I not need to check this test...?)

Reply:
This test is not used now. Thanks!